### PR TITLE
Adding cloud docs pages

### PR DESCRIPTION
**Pull request by 19sharath19**

### CODE REVIEW

File: src/data/nav-items.yaml [hunk 1]
```diff
@@ -41,12 +41,128 @@
...
+      path: /clouddocs/customizing/
+    # - title: "Managing Access Roles"
+    #   path: /clouddocs/access_management/
+    # - title: "Securing your data in Annotator for Clinical Data"
+    #   path: /clouddocs/data-security/
+    # - title: "Activity Tracker events"
+    #   path: /clouddocs/at_events/
+- title: Annotators
+  pages:
+    - title: "Allergies (Depracated)"
```
Comment (cagrev) [inline src/data/nav-items.yaml:78]: "Depracated" is misspelled here and a number of other places. Probably do a global search for that and correct it.

File: src/pages/clouddocs/customizing.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| Attribute Qualifiers | Configurations for detecting qualifying terms in the immediate context of an attribute and capturing the qualifiers as a field within the output attribute annotation. |
+| Ontological Relations | Ontology configurations for extracting ontological relations between concepts cooccurring within the same sentence. |
+| CPT Code Mapping | Mapping configurations for outputting CPT codes from the concept and procedure annotators. |
+`Table 1. Customizations facilitated by the configuaration editor`
+
+[Contact your IBM representative](https://www.ibm.com/account/reg/us-en/signup?formid=MAIL-watsonhealthna) to learn more about leveraging the configuration editor to customize the behavior of the service.
+
+## Cartridge Deployment
+
+1. The consumer uses the {{site.data.keyword.wh-acd_short}} Configuration Editor to create a new cartridge (or modify an existing one) and customizes the contents (artifacts) of the cartridge to their domain. After that, the consumer will **Export** the cartridge in order to save a snapshot of the cartridge.
```
Comment (cagrev) [inline src/pages/clouddocs/customizing.md:42]: "catridge" is misspelled in several places.

File: src/pages/clouddocs/annotator_concept_detection.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    "loincId": "MTHU020794",
+    "icd10Code": "S72.009?",
+    "nciCode": "C26794,C35153",
+    "snomedConceptId": "5913000,263225007",
+    "meshId": "M0010366",
+    "vocabs": "MTH,CHV,LNC,CSP,MSH,NCI,AOD,NCI_CTCAE,NDFRT,COSTAR,SNOMEDCT_US,DXP"
+}
+```
+
+Expanded detection will look for diseases, conditions, abnormalities, injuries, and procedures defined in the UMLS dictionary that ships with {{site.data.keyword.wh-acd_short}}.
```
Comment (cagrev) [inline src/pages/clouddocs/annotator_concept_detection.md:69]: Looks like there are a number of places where this isn't handled or formatted correctly. {{site.data.keyword.wh-acd_short}} Maybe do a global search for the double curly bracket to find them all and fix.

File: src/pages/clouddocs/annotator_smoking.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+|:--------------|--------|-------------|
+| Library | <ul><li>umls.latest</li><li>umls.2021AA</li><li>umls.2020AA</li><li>umls.2019AA <i>(deprecated - will be removed in 2022)</i></li></ul> | Defines the version of the UMLS library that is used when analyzing unstructured data. |
+
+The value `umls.latest` will reference the latest available version of UMLS within the service. As newer versions of UMLS are made available in the service, `umls.latest` library configurations will automatically leverage the latest available version of UMLS in the service once available. Declaration of a specific version of UMLS is recommended to avoid undesirable changes in output as newer versions of UMLS are made available within the service. Through declaration of a specific version of UMLS, newer versions of UMLS may be evaluated prior to use in production.
+
+### Annotation Types
+
+* aci.SmokingInd
+
+##$ aci.SmokingInd
```
Comment (cagrev) [inline src/pages/clouddocs/annotator_smoking.md:36]: Presentation/formatting issue with this one?

File: src/pages/clouddocs/customizing.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+    ```Curl
+  curl -X POST -u "apikey:{apikey}" \
+  --header “Content-Type: application/octet-stream” \
+  --header "Accept: application/json" \
+  --data-binary @./my_cartridges/name_of_cartridge_file.zip \
+  "{url}/v1/deploy?update=false&version=2018-01-17"
+```
+
+> Some large cartridge deployments can exceed the request timeout thresholds defined in the DataPower gateways (usually after 2 mins). In that event, you may receive the following error response. See [Cartridge Deployment Timeout](/docs/wh-acd?topic=wh-acd-troubleshoot#troubleshoot_deploy_timeout) for additional considerations during the deployment of large cartridges.
```
Comment (cagrev) [inline src/pages/clouddocs/customizing.md:101]: Accuracy and presentation/formatting issue? I think we're trying to convey something about an error that may surface, but not sure it applies here. Error example missing. If text is needed, it should wrap so it isn't scrolled off to the right.

File: src/pages/clouddocs/medical_codes.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
+
+---
+
+# Medical Codes -->
+
+Annotator for Clinical Data normalizes medical concepts into many common medical codes.  The following table shows the medical code support for each annotator available in Annotator for Clinical Data.
+
```
Comment (cagrev) [inline src/pages/clouddocs/medical_codes.md:23]: Is there a way to shrink the presentation of the tables so reader doesn't need to scroll off to the right? If not, can we have an indicator so reader knows there's more to scroll through? Not sure that's obvious as presented. I think you have to click into the table to realize it.

File: src/pages/clouddocs/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (cagrev) [inline src/pages/clouddocs/troubleshooting.md:1]: There's a monitoring and troubleshooting section. Wondering if this should be merged under that somehow? Maybe best to get content moved over and we can improve the flow/presentation of pages afterwards.

**Review by cagrev — CHANGES_REQUESTED** [review]

File: src/pages/clouddocs/annotator_concept_detection.md (hunk 3 above)
Comment (dcwebermn) [reply src/pages/clouddocs/annotator_concept_detection.md:69]: {{site.data.keyword.wh-acd_short}} should be replaced with IBM Clinical Data Annotator Service.

File: src/pages/clouddocs/customizing.md (hunk 2 above)
Comment (dcwebermn) [reply src/pages/clouddocs/customizing.md:42]: {{site.data.keyword.wh-acd_short}} should be replaced with IBM Clinical Data Annotator

File: src/pages/clouddocs/troubleshooting.md (hunk 7 above)
Comment (dcwebermn) [reply src/pages/clouddocs/troubleshooting.md:1]: I believe the clouddocs troubleshooting page can be removed however if we want to keep the content then ;yes it should be merged into the container edition troubleshooting section.

File: src/pages/clouddocs/medical_codes.md (hunk 6 above)
Comment (dcwebermn) [reply src/pages/clouddocs/medical_codes.md:23]: Unfortunately markdown syntax / rendering takes the largest column and normalizes all columns to that size which directly influences the size of the table.  We could try putting multiple word codes and the header in two text lines vs one to shrink the table size.

**Review by dcwebermn — CHANGES_REQUESTED** [review]
A few things to clean up yet - link references to cloud docs like this one need to be updated to reference pages in the container edition docs [Clinical Insights](/docs/wh-acd?topic=wh-acd-clinical_insights_overview#clinical_insights_overview).  

Curl examples that specify apikey and url need to be changed to use Bearer token and route_host like in this example
  ```Curl
  curl -X GET --header ‘Authorization: Bearer xxxxxxxxxxxxxxx’ \
  --header “Accept:application/json” \
  “<route_host>/v1/cartridges?version=2019-09-01"
  ```

**Review by dcwebermn — APPROVED** [review]
Updates approved.

File: src/data/nav-items.yaml (hunk 1 above)
Comment (dcwebermn) [reply src/data/nav-items.yaml]: resovled

File: src/pages/clouddocs/annotator_smoking.md (hunk 4 above)
Comment (dcwebermn) [reply src/pages/clouddocs/annotator_smoking.md]: resolved

**Pull request merged**